docker-compose-core.yaml

```diff
@@ -91,9 +91,28 @@ services:
     networks:
       - camunda-platform
 
+  kibana:
+    image: docker.elastic.co/kibana/kibana:${ELASTIC_VERSION:-7.17.0}
+    container_name: kibana
+    ports:
+      - 5601:5601
+    environment:
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
```

Comment:
I don't think this is working for Kibana, do we need to limit the heap size for kibana?

Reply:
TBH, I've copied this from a previous example. I don't think it's necessary